### PR TITLE
Add Php8 support 

### PR DESCRIPTION
**Pull request by gpupo**
Allow install at php 8

### CODE REVIEW

**Comment by gpupo** [comment]
@mghoneimy this repo is active ?

**Review by mghoneimy — COMMENTED** [review]
All looks good, just one question

**Comment by mghoneimy** [comment]
@gpupo apologies for having taken too long to take a look at this. Yep, this repo is still active!

**Comment by gpupo** [comment]
Sorry. Coverage clover will fail with php 8 but im refactory travis config to deal with that. Also add scripts session at composer config.

File: .travis.yml [hunk 1]
```diff
@@ -1,14 +1,19 @@
+sudo: false
```
Comment (mghoneimy) [inline .travis.yml:1]: We should refrain from using the `sudo` tag as it is deprecated as per https://github.com/IBM/portieris/issues/56

What do we need this for?

Comment (mghoneimy) [inline .travis.yml:1]: We should refrain from using the sudo tag as it is deprecated as per https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration

What do we need this for?

Comment (gpupo) [reply .travis.yml]: OK. Removed

**Pull request merged**